src/api/updateMessage.js

```diff
@@ -0,0 +1,15 @@
+import { apiPatch } from './apiFetch';
```

Comment:
Lets keep this file in this, as it has been reviewed. I'll make the api calls in another PR from next time!